src/open_inwoner/openzaak/tests/test_case_detail.py

```diff
@@ -467,7 +469,9 @@ def setUp(self):
                     "info_id": self.informatie_object_2["uuid"],
                 },
             ),
-            created=datetime.datetime(2021, 2, 12, 0, 0, 0),
+            created=datetime.datetime.strptime(
```

Comment:
It is safer to use native method for ISO and not mess with format strings: `datetime.fromisoformat()`